01-js-syntax/lab/01-echo-function.js

```diff
@@ -0,0 +1,7 @@
+function echo(param) {
+
```

Comment:
I don't think this solution needs a lot of reviews so I suggest deleting this file.